### PR TITLE
Allow user to progress through space creation & setup using Enter

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/17227

### CODE REVIEW

**Review by turt2live — APPROVED** [review]
a11y ftw

**Comment by turt2live** [comment]
your tests be unhappy though

File: src/components/structures/SpaceRoomView.tsx [hunk 1]
```diff
@@ -572,10 +581,17 @@ const SpaceSetupPrivateInvite = ({ space, onFinished }) => {
             ref={fieldRefs[i]}
             onValidate={validateEmailRules}
             autoFocus={i === 0}
+            onKeyDown={ev => {
+                if (ev.key === Key.ENTER) {
+                    ev.preventDefault();
+                    onClick();
+                }
+            }}
```
Comment (gsouquet) [inline src/components/structures/SpaceRoomView.tsx:589]: The enter key should by default work. I believe that if we were to wrap these fields on a `<form>` and set an `onSubmit` event listener on it, we would get that same "enter key" behaviour and other nice accessibility features that come from free thanks to the browser

File: src/components/structures/SpaceRoomView.tsx [hunk 2]
```diff
@@ -382,10 +384,17 @@ const SpaceSetupFirstRooms = ({ space, title, description, onFinished }) => {
             value={roomNames[i]}
             onChange={ev => setRoomName(i, ev.target.value)}
             autoFocus={i === 2}
+            onKeyDown={ev => {
+                if (ev.key === Key.ENTER) {
+                    ev.preventDefault();
+                    onClick();
+                }
+            }}
```
Comment (gsouquet) [inline src/components/structures/SpaceRoomView.tsx:392]: Same for this block

**Pull request merged**